### PR TITLE
Sprite Improvements (Blue Star and Reddit)

### PR DESCRIPTION
**Pull request by Kitsunemitsu**
## About The Pull Request
Replaces the blue star with a thousand legged version
Reddit now doesn't constantly do his animation

## Why It's Good For The Game
Nicer sprites!

## Changelog
:cl:
imageadd: added some new, nicer sprites
/:cl:

### CODE REVIEW

**Review by EgorDinamit — CHANGES_REQUESTED** [review]
Adjust pixel_x & base_pixel_x for blue star to be -16 so it's centered.
What's up with removal of all-around-helper's animation?

**Comment by Kitsunemitsu** [comment]
It's not a removal, it's just a lot slower.

**Comment by EgorDinamit** [comment]
> It's not a removal, it's just a lot slower.

But why? :thinking:

**Comment by Kitsunemitsu** [comment]
Because it kinda looks hyperactive

File: code/modules/mob/living/simple_animal/abnormality/aleph/blue_star.dm [hunk 1]
```diff
@@ -3,7 +3,8 @@
 	desc = "Floating heart-shaped object. It's alive, and soon you will become one with it."
 	health = 4000
 	maxHealth = 4000
-	icon = 'ModularTegustation/Teguicons/tegumobs.dmi'
+	pixel_x = -16
```
Comment (EgorDinamit) [inline code/modules/mob/living/simple_animal/abnormality/aleph/blue_star.dm:6]:
```suggestion
	pixel_x = -16
	base_pixel_x = -16
```
So it doesn't get reset when animating.

**Pull request merged**